packages/react/src/drafts/IssueLabel/getColorFromHex.ts

```diff
@@ -0,0 +1,138 @@
+import {getContrast} from 'color2k'
+import {Hsluv} from 'hsluv'
+
+/**
+ * transforms a hex color provided by the user into a color object with background and text colors
+ * @param colorHex — the hex color provided by the user
+ * @param colorScheme — the color scheme the user has currently set
+ * @param isSelected
+ * @param bgColor — the background color from the selected theme, needed to calc the contrast for the colors
+ * @returns
+ */
+export const getColorsFromHex = (
+  colorHex: string,
```

Comment:
Would it help to have a type like ``type Hex = `#${string}`;`` to help with inputs to the function? Could then remove the validation we're doing at the beginning and then the function could just return a value.

Definitely isn't true type safety but seems to mirror the rough checking we're doing below.